api/src/dto/ContentDto.ts

```diff
@@ -98,4 +107,9 @@ export class ContentDto extends _contentBaseDto {
     @IsString()
     @Expose()
     parentTagType?: TagType;
+
+    @IsOptional() // Optional as it is set upon change request processing
+    @IsBoolean()
+    @Expose()
+    publishDateVisible?: boolean;
```

Comment:
This is perhaps debatable, but as this field is replicated from the parent post/tag document, it should perhaps be renamed to `parentPublishDateVisible` to keep it in line with other replicated fields?